server/utils/applications/reportDataFromApplication.ts

```diff
@@ -34,14 +34,14 @@ const dutyToReferSubmissionDateFromApplication = (application: Application): str
 }
 
 const dutyToReferLocalAuthorityAreaNameFromApplication = (application: Application) => {
+  if (!isDutyToReferSubmittedFromApplication(application)) {
+    return ''
+  }
+
   const dutyToReferLocalAuthorityAreaName: string = (application.data as Record<string, unknown>)?.[
     'accommodation-referral-details'
   ]?.['dtr-details']?.localAuthorityAreaName
 
-  if (!dutyToReferLocalAuthorityAreaName) {
```

Comment:
Suggestion, shall we keep this validation when DTR is selected?(Same as existing code 'dutyToReferSubmissionDate'). I believe the LA name is mandatory if DTR section is selected

Reply:
There's another ticket to make it mandatory, so right now this should go -- and will be reinstated when that other ticket is addressed!

Reply:
make sense.